LaTeX/LaTeX.sublime-syntax

```diff
@@ -634,16 +634,32 @@ contexts:
         4: variable.parameter.function.latex
         5: punctuation.definition.group.brace.end.tex
       push: block-math-begin-end-command-body
+    - match: |-
+        (?x)
+        ((\\)begin)
+        \s* (\{) \s* ( IEEEeqnarray\*? ) \s* (\})
+        \s* (\{) \s* ( {{letter}}+ ) \s* (\})
+      # scope: meta.environment.math.block.latex
```

Comment:
We should avoid comments like this to not cause confusion when looking at them later. Seeing how the meta scope is pushed as a `meta_content_scope` in `block-math-begin-end-command-body`, I suppose we don't need this on the match here. If we wanted it to be included, changing that to `meta_scope` should suffice.